addons/mail/models/discuss/discuss_channel.py

```diff
@@ -94,7 +95,8 @@ def _generate_random_token(self):
     sfu_server_url = fields.Char(groups="base.group_system")
     rtc_session_ids = fields.One2many('discuss.channel.rtc.session', 'channel_id', groups="base.group_system")
     call_history_ids = fields.One2many("discuss.call.history", "channel_id")
-    is_member = fields.Boolean("Is Member", compute="_compute_is_member", search="_search_is_member", compute_sudo=True)
+    is_member = fields.Boolean("Is Member", compute="_compute_is_member", compute_sql="_compute_sql_is_member", compute_sudo=True)
+    is_member_with_parent = fields.Boolean(compute="_compute_is_member_with_parent", compute_sql="_compute_sql_is_member_with_parent", compute_sudo=True)
```

Comment:
What is the need for this new field?
How is it related to re-writing `_compute_is_member`?